src/lib/components/workspace/members/Tile.svelte

```diff
@@ -38,21 +47,21 @@
   let isMemberInfoPopup = false;
   let isMemberOwnershipPopup = false;
 
-  const handleMemberPopUpCancel = (flag) => {
+  const handleMemberPopUpCancel = (flag: boolean): void => {
```

Comment:
convert this 4 functions into one and just pass additional param with flag to set the respective variable as true false.